version-changelog.js

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env node
+
+var readline = require('readline');
+var fs = require('fs');
+var childProcess = require('child_process');
+
+var rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+var pluginName = process.argv[2];
+var pluginSlug = pluginName.substr(3);
+var mainPHPFile = pluginName + '/' + pluginName + '.php';
```

Comment:
In my plugin I'm keeping the DB version separated from the plugin version, because I wanted to update the DB only when it's actually needed.
I've just noticed that I've always forgot to update the version though, so it might be better to update it always.
What do you think?